discord/guild.py

```diff
@@ -1169,6 +1170,51 @@ def _create_channel(self, name, overwrites, channel_type, category=None, **optio
 
         return result
 
+    async def fetch_emojis(self):
+        """|coro|
+
+        Retrieves all custom :class:`Emoji`s from the guild.
+
+        Raises
+        ---------
+        HTTPException
```

Comment:
Missing documentation for `Forbidden`.

Reply:
I'm a bit confused here. In what cases would it return `Forbidden`? (Same for the other.)